website/docs/r/ec2_traffic_mirror_filter.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "EC2"
+layout: "aws"
+page_title: "AWS: aws_ec2_traffic_mirror_filter"
+description: |-
+  Provides an Traffic mirror filter
+---
+
+# Resource: aws_ec2_traffic_mirror_filter
+
+Provides an Traffic mirror filter.  
+Read [limits and considerations](https://docs.aws.amazon.com/vpc/latest/mirroring/traffic-mirroring-considerations.html) for traffic mirroring
+
+## Example Usage
+
+To create a basic traffic mirror filter
+
+```hcl
+resource "aws_ec2_traffic_mirror_filter" "foo" {
+  description                                 = "traffic mirror filter - terraform example"
+  network_services                            = ["amazon-dns"]
```

Comment:
Nit: Excessive whitespace to the left of the equals signs here and in below similar example configurations.